app/templates/components/ui-kit/image.hbs

```diff
@@ -0,0 +1,24 @@
+{{#if this.svg}}
+  <UiKit::Box
+    @display={{this.display}}
+    @margin={{this.margin}}
+    @height={{this.height}}
+    @width={{this.width}}
+  >
+    <SvgImage
+      @name={{this.svg}}
+      @alt={{this.alt}}
```

Comment:
If these are just arguments that passthrough to the child component, then documentation recommends the following syntax:
```suggestion
      @name={{@svg}}
      @alt={{@alt}}
```
I'm not sure if it has any differences, but at least visually it indicates that those are external attributes.